src/base/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-find_package(ZLIB 1.2.5.2 REQUIRED)
+find_package(Qt5Zlib ${requiredQtVersion})
```

Comment:
One last question, is it really preferable to use built-in Qt5Zlib, even when system ZLIB exists?
I suppose this code should do the inverse: check for ZLIB first, if not found then check for Qt5Zlib. Does that make sense?

Reply:
Additionally, I think it should be possible to force whichever is not the default by exposing a feature.